framework/src/security/security_auth.c

```diff
@@ -235,10 +235,6 @@ security_error auth_get_ecdsa_signature(security_handle hnd, security_ecdsa_para
 		SECAPI_HAL_RETURN(hres);
 	}
 
-	sign->data = (unsigned char *)malloc(h_sign.data_len);
```

Comment:
Why did you remove this malloc? Is memory allocated to sign->data inside SECAPI_DATA_DCOPY?